src/js/hmrc/README.md

```diff
@@ -11,5 +11,6 @@ Here are some of HMRC's fraud prevention headers required to be sent by web appl
 1. Gov-Client-Browser-Plugins (A list of browser plugins on the originating device)
 1. Gov-Client-Browser-Do-Not-Track (Indicates whether 'Do Not Track option' is enabled on the browser)
 1. Gov-Client-Local-IPs (A list of all local IPv4 and IPv6 addresses available to the originating device)
+2. Gov-Client-Local-IPs-Timestamp (A timestamp to show when Gov-Client-Local-IPs is collected)
```

Comment:
Please change `2` to `1`. GitHub Markdown automatically increments the number when processing the raw text. Otherwise, looks good and we'll be able to merge once that change is complete

Reply:
Done